x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/pipeline_select.test.tsx

```diff
@@ -0,0 +1,176 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { setMockActions, setMockValues } from '../../../../../__mocks__/kea_logic';
+
+import React from 'react';
+
+import { mount, shallow } from 'enzyme';
+
+import { EuiSelectable } from '@elastic/eui';
+import { euiThemeVars } from '@kbn/ui-theme';
+
+import { PipelineSelect } from './pipeline_select';
+
+const DEFAULT_VALUES = {
+  addInferencePipelineModal: {
+    configuration: {
+      pipelineName: '',
+    },
+  },
+  existingInferencePipelines: [],
+};
+
+const MOCK_ACTIONS = {
+  selectExistingPipeline: jest.fn(),
+};
+
+const EUI_BASE_SIZE = parseFloat(euiThemeVars.euiSize);
```

Comment:
@TattdCodeMonkey Is this the correct corollary for `euiTheme.base`? I think it is, but would like to confirm.